### DISCUSSION TITLE
Condition in Table

### DISCUSSION THREAD

**Discussion by leuedaniel** [Budibase/budibase#14010] [Help] [answered]
I have just noticed that when I want to create conditions for a table, I cannot select the data of the table itself.
For example, if I want to display a button when the value of the row in column X = y, I cannot select column X anywhere. Is this the case with the new table?

**Comment by mikesealey** [comment, marked as answer]
This may be a case for the [SuperTable custom component ](https://github.com/poirazis/bb-component-SuperTable). 

You can read [more here](https://docs.budibase.com/docs/custom-plugin) about how to get started bringing in custom components.

**Reply by leuedaniel** [reply]
Yes, I am aware of that. But why do I have the options for conditions but can't use them?
